packages/web/src/scss/components/UNSTABLE_ProductLogo/README.md

```diff
@@ -0,0 +1,63 @@
+# UNSTABLE ProductLogo
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+The ProductLogo component is used to display the logo of the product.
+
+```html
+<div class="UNSTABLE_ProductLogo" aria-label="Logo of the product">
+  <!-- Logo go here -->
+</div>
+```
+
+## Inverted variant
+
```

Comment:
I'm not sure if it would be better to explicitly state that the inverted logo must be set by the user and the component only changes the background color.